crowbar_framework/app/models/node_object.rb

```diff
@@ -1360,7 +1360,7 @@ def disk_release(device, owner)
     if device
       crowbar_wall[:claimed_disks] ||= {}
 
-      unless disk_owner(device) == owner
+      if owner.empty? || disk_owner(device) != owner
```

Comment:
Oh, wait, it should also happen for readying. We unconditionally release, without checking if there was an owner or not.

If I look at the one true source (ie, the similar code in the cookbook), it's fine to call with an empty / nil owner.

Now what I don't get is why this was failing without the changed test :/